docs/user_manual/source/pma.rst

```diff
@@ -23,6 +24,7 @@ The PMA can be deconfigured by setting ``PMA_NUM_REGIONS=0``. When doing this, `
 Address range
 ~~~~~~~~~~~~~
 The address boundaries of a PMA region are set in ``word_addr_low/word_addr_high``. These contain bits 33:2 of 34-bit, word aligned addresses. To get an address match, the transfer address ``addr`` must be in the range ``word_addr_low <= addr < word_addr_high``.
+Memory regions not covered by any PMA regions are treated as main memory with bufferable, cacheable and atomic attributes set.
```

Comment:
Can this be updated to reflect  the agreed upon defaults (depending on PMA_NUM_REGIONS)?